java/core/src/main/java/sleeper/core/util/RateLimitUtils.java

```diff
@@ -18,12 +18,20 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+/**
+ * A utility class for handling rate limits.
```

Comment:
This might be clearer as something like "Utilities to avoid rate limits in a remote API."